src/common/Hero.ts

```diff
@@ -22,43 +26,76 @@ export interface HeroDropdownProps {
   }>
 }
 
-export interface HeroInfoboxProps extends HeroHeadingProps, HeroButtonProps {
+export interface HeroBackgroundImageProps {
```

Comment:
I originally wanted to decouple having the background image from the key highlights, especially since in Next theme we completely do not have the key highlights feature.